### DISCUSSION TITLE
[✨] A way to clone elements

### DISCUSSION THREAD

**Discussion by Dindaleon** [QwikDev/qwik-evolution#260] [Proposals For Qwik]
### Is your feature request related to a problem?

I am making a drag and drop interface for my clients and would like to have a way to clone elements and keep their reactivity.

### Describe the solution you'd like

I would like to have something like React with `React.cloneElement()`

### Describe alternatives you've considered

At the moment i am using the default `element.cloneNode(true)` but its reactivity is being lost.

**Comment by gioboa** [comment]
How did you solve this specific task?

**Comment by PatrickJS** [comment]
By reactivity you're talking about signals.

So you can clone the html and any QRL should work but when you run into signals then them embedded data becomes an issue. You can also use context and if you have an issue with keys you can use uuids. If you have examples of what you're doing then it would make it easier to understand.

**Comment by wmertens** [comment]
@Dindaleon can't you solve this using components? You know what you are dragging, so you can render it twice?

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
